src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -76,8 +75,13 @@ export default function * fetchResourceCache (city: string, language: string, fe
       }, {})
     )
 
-    database.addCacheEntries(resourceCache)
-    yield call(database.writeResourceCache)
+    let newResourceCache = resourceCache
+    if (dataContainer.resourceCacheLoaded()) {
+      const currentResourceCache = yield call(dataContainer.getResourceCache)
+      newResourceCache = {...resourceCache, ...currentResourceCache}
+    }
```

Comment:
Yeah, so we might want to add an `addCacheEntries` function in our container...
Overwriting the resourceCache doesn't make sense most of the time, because the files that have previously been downloaded, are still persisted...

Reply:
I agree, at the moment this would make sense. Do you think `addCacheEntries` should also create an empty ResourceCache if it's `null`?   
On the other hand, as soon as we do incremental updating we probably want to manipulate the ResourceCache here and set the new one manually.

Reply:
> I agree, at the moment this would make sense. Do you think addCacheEntries should also create an empty ResourceCache if it's null?

Yes would make sense.

> On the other hand, as soon as we do incremental updating we probably want to manipulate the ResourceCache here and set the new one manually

I'd say thats too far away...

Reply:
Done.